example_newrelic.ini

```diff
@@ -0,0 +1,210 @@
+# ---------------------------------------------------------------------------
```

Comment:
Can you put this file in `newrelic` subfolder under https://github.com/mage-ai/mage-ai/tree/master/integrations ?